Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM openjdk:8-slim
+FROM bardsoftware/papeeria:java8
 
 ARG COSMAS_VERSION=Cosmas-1.0-SNAPSHOT
 
-ADD ./build/distributions/${COSMAS_VERSION}.zip  ./${COSMAS_VERSION}.zip
+ADD ./build/distributions/${COSMAS_VERSION}.zip  /${COSMAS_VERSION}.zip
 
 RUN unzip -n ${COSMAS_VERSION}.zip
 
-WORKDIR ./${COSMAS_VERSION}/bin/
+RUN apt-get -qq update && apt-get -qqy install file
+
+RUN wget -qO- https://get.docker.com/ | sh
+
+RUN sudo usermod -aG docker $(whoami)
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
We do set JAVA_HOME in the base image, you don't need to set it here

Reply:
removed